pkgs/by-name/co/cortex-tools/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "cortex-tools";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "grafana";
+    repo = finalAttrs.pname;
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+GWUC+lnCn5Nw2WytSvW/UsIMmMelCCsnKdBCHuue24=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [
+    "cmd/benchtool"
+    "cmd/cortextool"
+    "cmd/e2ealerting"
+    "cmd/logtool"
+  ];
+
+  meta = {
```

Comment:
It looks like there are tests that are run in the repo CI
https://github.com/grafana/cortex-tools/blob/30599226b7becfc58856fd1adf02f838867f7bbf/.github/workflows/validate_pr.yml#L39-L40

Are these tests not relevant to the binaries built in this module? If they are, please add this to the `checkPhase`.

It would also be useful to have the `versionCheckHook`, as it is an option here
https://github.com/grafana/cortex-tools/blob/30599226b7becfc58856fd1adf02f838867f7bbf/cmd/cortextool/main.go#L41